lib/plugins/extension/style.less

```diff
@@ -281,6 +281,18 @@
  * Enabled/Disabled overrides
  */
 #extension__list {
+    
+    .enabled, .disabled,
+    .updatable {
+        display: none;
```

Comment:
Just found that this means that the "installed plugins" and "installed templates" of the extension manager are unusable without JavaScript.